### PR TITLE
add lang field in user model

### PR DESCRIPTION
**Pull request by willkoua**
| Q                                          | R
| ------------------------------------------ | -------------------------------------------
| Type of contribution ?                     | [Enhancement]
| Tickets (_issues_) concerned               | [TV-250]

---

##### What have you changed ?
add lang field in user model

##### How did you change it ?
...

##### Is there a special consideration?
...

Verification :
 -  [x] My branch name respect the standard defined in `CONTRIBUTING.md`
 -  [x] All my commits respect the standard defined in `CONTRIBUTING.md`
 -  [x] My additions are I18N
 -  [x] This Pull-Request fully meets the requirements defined in the issue
 -  [ ] I added or modified the attached tests
 -  [ ] I added or modified the documentation

[TV-250]: https://fjnr-inc.atlassian.net/browse/TV-250

### CODE REVIEW

File: blitz_api/models.py [hunk 1]
```diff
@@ -19,13 +19,19 @@
...
+    LANG_FR = 'fr'
+    LANG_EN = 'en'
 
     GENDER_CHOICES = (
         ('M', _("Male")),
         ('F', _("Female")),
         ('T', _("Trans")),
         ('A', _("Do not wish to identify myself")),
     )
+    LANG_CHOICE = (
```
Comment (RignonNoel) [inline blitz_api/models.py]: LANG_CHOICE**S**

File: blitz_api/models.py [hunk 2]
```diff
@@ -90,6 +96,12 @@ class User(AbstractUser):
         choices=GENDER_CHOICES,
         verbose_name=_("Gender"),
     )
+    lang = models.CharField(
```
Comment (RignonNoel) [inline blitz_api/models.py]: Should be "Language" to be clear

**Review by RignonNoel — CHANGES_REQUESTED** [review]
You did not change the tests to verifyif we can change this field on register and edit profile

File: blitz_api/models.py (hunk 1 above)
Comment (willkoua) [reply blitz_api/models.py]: done

File: blitz_api/models.py (hunk 2 above)
Comment (willkoua) [reply blitz_api/models.py]: done

**Review by RignonNoel — CHANGES_REQUESTED** [review]

File: blitz_api/models.py [hunk 3]
```diff
@@ -90,6 +96,12 @@ class User(AbstractUser):
         choices=GENDER_CHOICES,
         verbose_name=_("Gender"),
     )
+    language = models.CharField(
+        max_length=100,
+        choices=LANGUAGE_CHOICES,
+        verbose_name=_("Lang"),
```
Comment (RignonNoel) [inline blitz_api/models.py]: Lang -> Language

**Review by RignonNoel — CHANGES_REQUESTED** [review]
Aussi, tu avais dit que tu ferais une seule migration quand Jerome t'a demande

File: blitz_api/migrations/0022_auto_20191003_1254.py [hunk 4]
```diff
@@ -0,0 +1,23 @@
...
+            model_name='historicaluser',
+            old_name='lang',
+            new_name='language',
+        ),
+        migrations.RenameField(
+            model_name='user',
+            old_name='lang',
+            new_name='language',
+        ),
+    ]
```
Comment (Clara-Celle) [inline blitz_api/migrations/0022_auto_20191003_1254.py]: Penses à merger les deux migrations en une

**Review by Clara-Celle — CHANGES_REQUESTED** [review]

**Review by RignonNoel — APPROVED** [review]

**Pull request merged**